Gifski/Gifski.swift

```diff
@@ -219,7 +225,7 @@ final class Gifski {
 	private func imageGenerator(
 		for conversion: Conversion,
 		jobKey: String
-	) -> Result<(generator: AVAssetImageGenerator, times: [CMTime]), Error> {
+	) -> Result<(generator: AVAssetImageGenerator, times: [CMTime], fps: Int), Error> {
```

Comment:
I think this could just use `throws` instead of `Result`. I generally only use `Result` for async or when I need to preserve the Error type.

Reply:
I generally agree, but In this case, `Error` is the specialized `Gifski.Error` not the general `Swift.Error`. Switching it to a throwing method loses the specialized Error type that its caller expects.